build-tools/tasks/docs.js

```diff
@@ -24,7 +24,12 @@ function validatePublicFiles(definitionFiles) {
 }
 
 function componentDocs() {
-  const definitions = documentComponents(require.resolve('../../tsconfig.json'), 'src/*/index.tsx');
+  const definitions = documentComponents(require.resolve('../../tsconfig.json'), 'src/*/index.tsx', undefined, {
+    extraExports: {
+      FileDropzone: ['useFilesDragging'],
+      TagEditor: ['getTagsDiff'],
+    },
```

Comment:
New feature, if component exports something else beside default export and `ComponentProps` types, it needs to be declared here

This is needed to prevent "public by accident" exports